CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Release (2024-XX-XX)
 
+- `iaas`: [v0.5.0](services/iaas/CHANGELOG.md#v050-2024-07-22)
+  - **Feature**: New methods to manage networks for IPv6:
+    - `V1CreateNetworkIPv6`
+    - `V1UpdateNetworkIPv6`
```

Comment:
These are not new methods, right? They are the types of the new `Ipv6` field mentioned below

Reply:
removed the mentioned part